app/workers/fieri_notify_worker.rb

```diff
@@ -21,9 +21,25 @@ def perform(cookbook_version_id)
     data = {
       'cookbook_name' => cookbook_version.name,
       'cookbook_version' => cookbook_version.version,
-      'cookbook_artifact_url' => cookbook_version.tarball.url
+      'cookbook_artifact_url' => cookbook_artifact_url(cookbook_version)
     }
 
     response = Net::HTTP.post_form(uri, data)
   end
+
+  private
+
+  def cookbook_artifact_url(cookbook_version)
```

Comment:
Good question.

I don't *think* so, because you can only have all expiring urls with S3 or no expiring urls.  But let me verify.